<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiQmlForm</class>
 <widget class="QDialog" name="MultiQmlForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>206</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Assign style</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Layers:</string>
     </property>
     <property name="buddy">
      <cstring>lvMapLayers</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QRadioButton" name="rbnRasterLayers">
     <property name="text">
      <string>Raster</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QRadioButton" name="rbnVectorLayers">
     <property name="text">
      <string>Vector</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QListView" name="lvMapLayers">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pbnApplyStyle">
       <property name="text">
        <string>Apply style ...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbnRestoreDefaultStyle">
       <property name="text">
        <string>Restore initial style</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbnSelectAllLayers">
       <property name="text">
        <string>Select all layers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QCheckBox" name="checkMakeDefault">
     <property name="text">
      <string>Save as default</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pbnApplyStyle</tabstop>
  <tabstop>lvMapLayers</tabstop>
  <tabstop>pbnRestoreDefaultStyle</tabstop>
  <tabstop>pbnSelectAllLayers</tabstop>
  <tabstop>rbnRasterLayers</tabstop>
  <tabstop>rbnVectorLayers</tabstop>
  <tabstop>pbnClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
